icon-two-common-application-1-1/src/main/java/ca/bc/gov/open/icon/controllers/HealthController.java

```diff
@@ -35,6 +36,7 @@
 import org.springframework.ws.server.endpoint.annotation.RequestPayload;
 import org.springframework.ws.server.endpoint.annotation.ResponsePayload;
 
+@Profile("!test")
```

Comment:
what's this for?

Reply:
without that, the RabbitMQ will try to do a connection and throw error in test.  I do not know exactly how it works, I will ask Ethan as he asked me to add it.